src/components/answersList/styles.css

```diff
@@ -29,6 +31,17 @@
   top: 0;
 }
 
+.answers-list-table::-webkit-scrollbar {
+  width: 8px;
+  background-color: #f5f5f5;
+}
+
+.answers-list-table::-webkit-scrollbar-thumb {
+  border-radius: 16px;
+  background-color: rgb(255, 0, 0);
+  background-image: -webkit-linear-gradient(90deg, transparent, rgba(0, 0, 0, 0.3) 50%, transparent, transparent);
+}
```

Comment:
Very cool! Unfortunately it sometimes collides with `th`s.
![Screenshot from 2022-01-02 22-50-18](https://user-images.githubusercontent.com/47178494/147890401-a26820cc-faa2-4192-be5a-25054108f0f6.png)

Reply:
I've tested it on the same version of Chrome and those issues appeared only once when I scrolled list with mouse's wheel very slow. I asked Tomek to check it out, and he said that everything looks okay to him. I don't know what else I can try to fix it.